platform/security/certificate/src/main/java/org/codice/ddf/certificate/PEMManager.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.certificate;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.security.KeyPair;
+import java.security.PrivateKey;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import org.bouncycastle.asn1.pkcs.PrivateKeyInfo;
+import org.bouncycastle.cert.X509CertificateHolder;
+import org.bouncycastle.cert.jcajce.JcaX509CertificateConverter;
+import org.bouncycastle.openssl.PEMDecryptorProvider;
+import org.bouncycastle.openssl.PEMEncryptedKeyPair;
+import org.bouncycastle.openssl.PEMKeyPair;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter;
+import org.bouncycastle.openssl.jcajce.JcePEMDecryptorProviderBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+        /*+----------------------------------------------------------------------
+         ||
+         ||  Class: PEMManager
+         ||
+         ||        Purpose:  Collection of static methods that extracts objects from PEM files. Currently PEMKeyPair, PrivateKeys, and X509Certificate's.
+         ||
+         ||  Inherits From:  None
+         ||
+         ||     Interfaces:  None
+         ||
+         |+-----------------------------------------------------------------------
+         ||
+         ||      Constants:  None
+         ||
+         |+-----------------------------------------------------------------------
+         ||
+         ||  Class Methods:     X509Certificate     loadX509FromPEM(String)
+         ||                     PrivateKey          getPrivateKeyFromPEMFile(String, String)
+         ||
+         ||
+         ++-----------------------------------------------------------------------*/
+
+public class PEMManager {
```

Comment:
Acronyms should be camel-cased, i.e., PemManager.

Also, "Manager" is usually not a very good name for a class (too vague). Could we rename it something like PemFileReader, or just PemFile?